package.json

```diff
@@ -11,43 +11,45 @@
     "axios": ">= 0.18.0",
     "jquery": "^3.3.1",
     "lodash": "^4.17.11",
+    "planout": "git://github.com/rawls238/Planout.js.git#97807c149d01031b972d9c3df7fb93558a166120",
     "vue": "^2.6.8",
     "vue-template-compiler": "^2.6.8",
     "vue-slick": "^1.1.15"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.25.0",
-    "@typescript-eslint/parser": "^2.25.0",
-    "@typescript-eslint/typescript-estree": "^2.25.0",
-    "@types/jest": "^23.1.4",
+    "@types/jest": "23.1.4",
     "@types/jquery": "3.3.29",
-    "@types/lodash": "^4.14.121",
-    "@vue/cli-plugin-eslint": "^3.9.0",
-    "@vue/cli-plugin-typescript": "^4.4.1",
-    "@vue/cli-plugin-unit-jest": "^4.4.1",
-    "@vue/cli-service": "^4.4.1",
-    "@vue/eslint-config-standard": "^4.0.0",
-    "@vue/eslint-config-typescript": "^4.0.0",
-    "@vue/test-utils": "^1.0.0-beta.29",
-    "axios": "^0.18.0",
-    "babel-eslint": "^10.0.1",
-    "eslint": "^6.8.0",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-no-only-tests": "^2.4.0",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "eslint-plugin-vue": "^6.2.2",
+    "@types/lodash": "4.14.121",
+    "@typescript-eslint/eslint-plugin": "2.25.0",
+    "@typescript-eslint/parser": "2.25.0",
+    "@typescript-eslint/typescript-estree": "2.25.0",
+    "@vue/cli-plugin-eslint": "4.2.3",
```

Comment:
Why are we downgrading the vue CLI + plugins?

Reply:
This combination removed all warnings.